packages/composite-checkout/package.json

```diff
@@ -43,8 +43,7 @@
 		"@wordpress/i18n": "^3.20.0",
 		"@wordpress/react-i18n": "^1.0.3",
 		"debug": "^4.1.1",
-		"prop-types": "^15.7.2",
-		"react-stripe-elements": "^4.0.2"
```

Comment:
We should add a dependency on `@stripe/stripe-js';`, since we import stuff from it in this package :) I'm not sure if it needs to be a normal or dev dependency since we're just importing types?

The reason for adding it would be to make sure that yarn resolves the correct version of the package to import.

Reply:
> since we import stuff from it in this package

do we? 🤔 I think all instances of those imports have been removed.

Reply:
Oh, you're right! I actually meant the stuff in the `wpcom-checkout` package, like this: https://github.com/Automattic/wp-calypso/pull/56089/files?file-filters%5B%5D=.json&file-filters%5B%5D=.tsx#diff-9d61c8c75da2cf3dc7f71a02372530c4ab084ea35fd215e2dcaecf9f0f100cc9R10

Do we need to update the package.json for that package?

Reply:
Oh! good point. I didn't consider that when I updated the type imports. I pushed an update to include `@stripe/stripe-js` in the `wpcom-checkout` package.